### DISCUSSION TITLE
no save-games found on usb drive

### DISCUSSION THREAD

**Discussion by eadmaster** [bucanero/apollo-ps4#142] [Q&A] [answered]
I have a bunch of savegames exported in zip format with ast on a usb pendrive.

I can read and copy the saves just fine on a pc, but when trying to import them back i get the "no save-games found" message.

Do i have to extract them to  have them listed in AST?

**Comment by bucanero** [comment]
> I have a bunch of savegames exported in zip format with ast on a usb pendrive.
> 
> I can read and copy the saves just fine on a pc, but when trying to import them back i get the "no save-games found" message.
> 
> Do i have to extract them to have them listed in AST?

Correct, zipped saves need to be decompressed so Apollo can import/copy them back to your system. You'll have to decompress and copy to your USB drive, keeping this folder structure: `/PS4/APOLLO/<save folder>/`

so once it's unpacked it should look something like 
```
/PS4/APOLLO/my_save_dir/
/PS4/APOLLO/my_save_dir/savedata.bin
/PS4/APOLLO/my_save_dir/sce_sys/
/PS4/APOLLO/my_save_dir/sce_sys/param.sfo
/PS4/APOLLO/my_save_dir/sce_sys/icon0.png
...etc...
```

Then just open Apollo and open the "USB saves" menu. Saves will be listed and you can copy to HDD.

**Comment by eadmaster** [comment]
Ok, what is the expected name for `my_save_dir` if you have multiple saves?

I am not sure if to use the actual zip filename, the  title id/name, or something else.

(Just provide a real full path example, and it should be obvious for me what to use.)

**Reply by bucanero** [reply, marked as answer]
`my-save-dir` is anything you like, it has no requirement. Just keep one folder for each save, that's the only thing to keep in mind. Apollo will load details about title-id, game name, etc++ from the `param.sfo` file, so the folder has no restriction.

I suggest avoiding spaces and non-regular english characters for folders, as I never tested edge cases.

If you want examples: just use Apollo, open "HDD saves", select any save and click "copy save to USB". Then you'll see how Apollo creates the folder for that save. But as I said it's just an example. Any folder should work.
